From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 25 Aug 2020 17:08:17 +0100
Subject: configs: Add CONFIG_SPS30=m

Enable the driver for the Sensirion SPS30 particulate matter sensor.
The i2c-sensor overlay already has support for the device, but it does
nothing without also having the driver module.

See: https://github.com/raspberrypi/firmware/issues/1460

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 170d0870a3b0..c6d146029bad 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -1316,6 +1316,7 @@ CONFIG_MCP320X=m
 CONFIG_MCP3422=m
 CONFIG_TI_ADS1015=m
 CONFIG_BME680=m
+CONFIG_SPS30=m
 CONFIG_DHT11=m
 CONFIG_HDC100X=m
 CONFIG_HTU21=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index dbeeff9c2a5d..c4620ef40ebd 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -1352,6 +1352,7 @@ CONFIG_MCP320X=m
 CONFIG_MCP3422=m
 CONFIG_TI_ADS1015=m
 CONFIG_BME680=m
+CONFIG_SPS30=m
 CONFIG_DHT11=m
 CONFIG_HDC100X=m
 CONFIG_HTU21=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 41265fbe8ab2..9789478cc4d1 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -1322,6 +1322,7 @@ CONFIG_MCP320X=m
 CONFIG_MCP3422=m
 CONFIG_TI_ADS1015=m
 CONFIG_BME680=m
+CONFIG_SPS30=m
 CONFIG_DHT11=m
 CONFIG_HDC100X=m
 CONFIG_HTU21=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index e8889a89b3c5..e75e32b63bda 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1344,6 +1344,7 @@ CONFIG_MCP320X=m
 CONFIG_MCP3422=m
 CONFIG_TI_ADS1015=m
 CONFIG_BME680=m
+CONFIG_SPS30=m
 CONFIG_DHT11=m
 CONFIG_HDC100X=m
 CONFIG_HTU21=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index f278668e178d..a69ed1899039 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -1197,6 +1197,7 @@ CONFIG_IIO_BUFFER_CB=m
 CONFIG_MCP320X=m
 CONFIG_MCP3422=m
 CONFIG_BME680=m
+CONFIG_SPS30=m
 CONFIG_DHT11=m
 CONFIG_HTU21=m
 CONFIG_APDS9960=m
